_includes/templates/nav-about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
```

Comment:
It would be the same functionality we have implemented on the bureau page when you click the play button.